docs/pages/reference/js/dscvr/getDscvrFrameMessage.mdx

```diff
@@ -0,0 +1,27 @@
+# getDscvrFrameMessage
+
+Returns a `DscvrFrameMessageReturnType` object from the message payload. Throws an error if the message is invalid. Requires the `@dscvr/frames-validator` peer dependency to be installed.
```

Comment:
```suggestion
Returns a `DscvrFrameMessageReturnType` object from the message payload. Throws an error if the message is invalid. Requires the `@dscvr-one/frames-validator` peer dependency to be installed.
```

Reply:
Thanks done (it was frames-adapter BTW lol)